kubernetes/redpanda/values.yaml

```diff
@@ -0,0 +1,84 @@
+# Default values for redpanda.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 3
+
+image:
+  repository: vectorized/redpanda
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+redpanda:
+  advertised_rpc_api:
+    port: 33145
+  advertised_kafka_api:
+    port: 9092
+  admin:
+    port: 9644
+  smp: 1
+
+volumeMount:
+  name: config
+  mountPath: /tmp/config 
+  base:
+    name: base-config
+    mountPath: /tmp/base-config
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: false
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations: {}
+
+podSecurityContext: {}
```

Comment:
Seems unused, should be added to podSpec.

Reply:
Done